### DISCUSSION TITLE
Index file for combined Roslin genome/mito sequence

### DISCUSSION THREAD

**Discussion by yaaminiv** [RobertsLab/resources#1166] [Q&A] [answered]
Earlier I combined the Roslin genome + mitochondrial sequence from NCBI (file [here](https://gannet.fish.washington.edu/spartina/project-oyster-oa/Haws/data/cgigas_uk_roslin_v1_genomic-mito.fa)). I want to look at my data in IGV, but need a .fai for the genome! How do I create an index file? 

@sr320 Do you already have a .fai file for the combined genome somewhere?

**Comment by yaaminiv** [comment]
Assuming I could use [`samtools faidx`](http://www.htslib.org/doc/samtools-faidx.html) to create the index file

**Comment by sr320** [comment]
Working on putting this in cloud as we speak... but I would you index feature in IGV

**Reply by yaaminiv** [reply]
so create the index myself in IGV?

**Comment by sr320** [comment]
Yes, that is what I suggest


>

**Comment by yaaminiv** [comment, marked as answer]
Used instructions from IGV to index FASTA and create a genome for IGV: https://software.broadinstitute.org/software/igv/LoadGenome

combined Roslin/mito genome, index file, and IGV genome found here: https://gannet.fish.washington.edu/spartina/project-oyster-oa/data/Cg-roslin/
